content/docs/configuration-system/content.md

```diff
@@ -0,0 +1,220 @@
++++
+fragment = "content"
+weight = 100
+title = "Configuration System"
+subtitle = "knoxite's configuration system"
+
+[sidebar]
+  sticky = true
++++
+
+<p>
+
+Knoxite provides a configuration system for your repositories. You can declare
+shorthands to your repositories and provide default values for settings like
+encryption, compression or excludes.
+The `config` command offers a variety of subcommands in order help you to manage
+your configuration.
+
+A configuration is structured in [json](https://json.org) format and does look
+like this:
+```json
+{
+    "repositories": {
```

Comment:
isn't this meant to be an array of repositories?

Reply:
These are a bunch of json objects